src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
@shehi Your PR was not rejected. On the contrary, the fact that Taylor asked you to fix the whitespace to match his style indicates that he wants to accept this change.

The [laravel docs](http://laravel.com/docs/5.0/contributions#coding-style) have some *very* basic coding guidelines, but it can't possibly cover everything. Anything that is not covered there, can be understood from reading Laravel's source code. If anyone sends a PR where the format does not fully match Laravel's coding style, it is kindly pointed out to them, they change it, and then the PR can be accepted.

Also, keep in mind that Taylor is just one person and he can't possibly get to everything himself. @GrahamCampbell has been generously giving much of his own time to help Taylor out with fielding issues and pull requests, so that Taylor can focus on the actual code.

---

**P.S.** you don't have to submit another PR. You can update this one. After you make your changes, run this command:

```
git commit -a --amend --no-edit
git push -f origin patch-1
```

This will update the current PR with the changes you've made.